CMSgov/easi-app
# [EASI-2906] TRB optional collaboration groups

# EASI-2906

## Changes and Description

- Updated schema to make collaboration group checkboxes optional
- Fixed bug in collab groups question - subfields were not being cleared if form was reloaded
- Added "no answer selected" text to review component if no collab groups are selected



## How to test this change

1. Create TRB request and navigate to request form
2. Confirm that collaboration group checkboxes are optional on first step
3. Confirm that subfield inputs are cleared when parent checkbox is unchecked
    - Answer will not clear from form UI unless page is refreshed
4. Confirm "no answer selected" text displays in review step



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
